hadoop-tools/hadoop-distcp/src/main/java/org/apache/hadoop/tools/mapred/CopyCommitter.java

```diff
@@ -152,6 +152,15 @@ public void abortJob(JobContext jobContext,
   }
 
   private void cleanupTempFiles(JobContext context) {
+    final boolean directWrite = context.getConfiguration().getBoolean(
```

Comment:
There is an existing call to `Configuration conf = context.getConfiguration();` below. Can you please lift that up and then reuse it for the `conf.getBoolean(...)` calls that you need to add?

Reply:
done